heron/scheduler-core/src/java/com/twitter/heron/scheduler/SchedulerMain.java

```diff
@@ -351,6 +353,15 @@ public boolean runScheduler() {
       // TODO(mfu): timeout should read from config
       SchedulerStateManagerAdaptor adaptor = new SchedulerStateManagerAdaptor(statemgr, 5000);
 
+      // get a packed plan and schedule it
+      PackingPlans.PackingPlan serializedPackingPlan = adaptor.getPackingPlan(topology.getName());
+      if (serializedPackingPlan == null) {
+        LOG.severe("Failed to obtain a valid PackingPlan from the state manager");
```

Comment:
I think a best practices doc could be good. We're already starting with some of those in #1311. Generally I think it's good to include identifying info in the log messages, in case this is logged in a shared env or from a suite of unit or integration tests. That said, some discretion is required, since I could also see that practice becoming overkill.